### DISCUSSION TITLE
Returning more than one data cascade element in eleventyComputed.js

### DISCUSSION THREAD

**Discussion by crockwave** [11ty/eleventy#1489] [Uncategorized]
I am having difficulty generating more than one `eleventyComputed` data object into the data cascade.  Is there a method where I can generate more than one of these data objects?

I am using `eleventyComputed.js` in the global data folder, to add data extracted from a `json` file also in the global data folder. It extracts the `.feed.entry` portion of the json object into the data cascade. The code looks like this:

```
module.exports = {
  eleventyComputed: {
    myCopyrights: data => {
      return data.copyrights.feed.entry;
    }
  }
};
```
I want to also add a myLinks data object to the data cascade.  When I make the code in `eleventyComputed.js` look like the following, it only adds the last data object to the data cascade, but not both.

```
module.exports = {
  eleventyComputed: {
    myCopyrights: data => {
      return data.copyrights.feed.entry;
    }
  }
};
module.exports = {
  eleventyComputed: {
    myLinks: data => {
      return data.links.feed.entry;
    }
  }
};
```

**Comment by pdehaan** [comment]
I believe your second `module.exports` would overwrite your first object.
Try something like:

```js
module.exports = {
  eleventyComputed: {
    myCopyrights: data => {
      return data.copyrights.feed.entry;
    },
    myLinks: data => {
      return data.links.feed.entry;
    }
  }
};
```

**Reply by crockwave** [reply]
That works.  Thank you very much.  I tried that previously, but neglected to include the comma between the two commands
